pkg/authz/authz.go

```diff
@@ -60,3 +60,8 @@ type Authorization interface {
 	Authorize(ctx context.Context, pr PolicyReq) error
 	AuthorizePAT(ctx context.Context, pr PatReq) error
 }
+
+// CallBack send auth request to an external service.
+type CallBack interface {
+	Authorize(ctx context.Context, pr PolicyReq) error
```

Comment:
Usually, interfaces with a single method are named after the method (Read - Reader, Write - Writer...), so this interface should be `Authorizer`.